Automattic/studio
# Studio: Add dynamic code block to execute wp-cli commands UI AI assistant

Related to https://github.com/Automattic/dotcom-forge/issues/7526

## Proposed Changes

This PR adds that dynamic code block that allows to execute CLI commands inside that block. It also does the following:

* if the code block contains more than 1 CLI command, the Run button is disabled
* the `Run` button is disabled for code blocks that don't have CLI commands
* when you execute the CLI command, see the output, navigate to another tab and come back, you can see the results of the output preserved 

## Testing Instructions



* Pull the changes from this branch locally
* Start the app with `STUDIO_AI=true npm start`
* Navigate to the Assitant tab
* Ask the assistant to give you a CLI command to list users
* Observe that it produces a code block
* Run the command with `Run` button 
* Confirm that you see the output being rendered
* Navigate to a different tab
* Come back to the AI Assistant
* Observe that the output of the CLI command is preserved:

<img width="872" alt="Capture d’écran, le 2024-06-13 à 10 26 19" src="https://github.com/Automattic/studio/assets/25575134/e47d7aea-3299-4a82-9754-07948eda75c9">

* Try asking for different CLI commands and checking the output

## Pre-merge Checklist
